receiver/hostmetricsreceiver/internal/scraper/filesystemscraper/filesystem_scraper_test.go

```diff
@@ -119,14 +119,16 @@ func TestScrapeMetrics(t *testing.T) {
 			expectedDeviceDataPoints: 2,
 			expectedDeviceLabelValues: []map[string]string{
 				{
-					"device":      "device_a",
-					"mount.point": "mount_point_a",
-					"type":        "fs_type_a",
+					"device":     "device_a",
+					"mountpoint": "mount_point_a",
+					"type":       "fs_type_a",
+					"mode":       "unknown",
```

Comment:
Minor point: I didn't notice this before but I wonder if it would be better to not add the label at all if `mode` is `unknown` (same for any other label where the value is unknown)? I'm not sure if the specification has any rule / guideline on how to handle this kind of case.

Reply:
@james-bebbington - I think that's a fair point. On the contrary though, I think this will help differentiate between cases where an expected label was not collected due to some reason vs the case where certain label(s) do not exist on an MTS because it's an aggregate of another measurement. That said, if we already have an agreed upon guideline to handle such cases, happy to incorporate that. @bogdandrutu @tigrannajaryan do you guys know if we have something in the spec to handle such cases?

Reply:
Not that I know of. Good opportunity to clarify that.